example.php

```diff
@@ -177,6 +177,9 @@ function my_theme_register_required_plugins() {
 				'The following recommended plugins are currently inactive: %1$s.',
 				'theme-slug'
 			),
+			'notice_force_activation'         => _n_noop(
+				'The following plugin has been automatically activated because it is required by the current theme: %s', 'The following plugins have been automatically activated because they are required by the current theme: %s'
+			),
```

Comment:
`_n_noop()` needs the text-domain as the last argument. For the example file this is `theme-slug`.